brave/brave-ios
# Fix #7233: Wallet Tx Activity Supports multiple networks

## Summary of Changes
Currently, all the screens that display transaction information is restricted by the default network from the current user selected account's coin type. Core now removes this restriction, so front-end is now able to fetch tx for all networks.


This pull request fixes #7233 #7385

## Submitter Checklist:


## Test Plan:

1. Open Brave browser and login to Brave Wallet
2. make some transaction using the same coin type but in different networks. (For example: Using a solana account to make some tx on Solana Mainnet and Solana Devnet. Using a eth account to make some tx on Eth mainnet and Goerli)
3. Confirm or reject those tx
4. Go to Activity tab
5. Observe that all non-rejected for all accounts in all networks will be displayed 
6. Go to each account screen 
7. Observe that all tx made by this account in all networks will be displayed 
8. Go to activity history from the wallet panel 
9. Observe that all tx made by this account in all networks will be displayed 
10. Click the global BSS button and choose a different network.
11. Repeat from step 4
12. Step 5, 7 and 9 should also be expected. 


## Screenshots:

https://github.com/brave/brave-ios/assets/1187676/7c750d9c-f8fa-44b4-a835-14ebbb13aff4

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
